src/v2/Apps/Partner/Components/ScrollToPartnerHeader/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import { ScrollIntoView } from "v2/Utils"
+import { Media } from "v2/Utils/Responsive"
+import { MOBILE_NAV_HEIGHT, NAV_BAR_HEIGHT } from "v2/Components/NavBar"
+
+export const ScrollToPartnerHeader: React.FC = ({ children }) => {
+  return (
+    <>
+      <Media greaterThan="xs">
+        <ScrollIntoView
+          selector="#jumpto--PartnerHeader"
+          offset={NAV_BAR_HEIGHT}
+        >
+          {children}
+        </ScrollIntoView>
+      </Media>
+      <Media at="xs">
+        <ScrollIntoView
+          selector="#jumpto--PartnerHeader"
+          offset={MOBILE_NAV_HEIGHT}
+        >
+          {children}
+        </ScrollIntoView>
+      </Media>
+    </>
```

Comment:
Since any click will happen on the client of course we don't need to hack around the offset with rendering with media

Reply:
I use Media here to set the right offset. I can use useMatchMedia hook, is it okay here?

Reply:
Yeah totally fine in this case